packages/render-fidelity-tools/test/renderers/model-viewer/index.html

```diff
@@ -55,23 +55,19 @@
   <script>
     harness.addEventListener('scenario-change', () => {
       const {scenario} = harness;
-
-      modelViewer.addEventListener('model-visibility', (event) => {
-        if (event.detail.visible) {
-          //TODO: this is a temporary workaround for taking screenshots before the model has been loaded
-          //This will be fixed when the screenshot function is refactored.
-          setTimeout(()=>{
-            self.dispatchEvent(new CustomEvent('model-ready'));
-          }, 500);
-        }
-      }, { once: true });
+      modelViewer.addEventListener('poster-dismissed', ()=>{
+        this.dispatchEvent(new CustomEvent('model-ready'));
+      }, {once: true});
       
       modelViewer.style.width = `${scenario.dimensions.width}px`;
       modelViewer.style.height = `${scenario.dimensions.height}px`;
 
       modelViewer.style.backgroundColor = "rgba(255,255,255,0)";
 
       modelViewer.src = scenario.model;
+
+      modelViewer.skyboxImage = null;
+      modelViewer.environmentImage = null;
```

Comment:
I would put these inside the else statement instead; that way when you change scenarios but have the same lighting, it doesn't switch back and forth. Realistically this is probably optimized out by lit-element's batched updates anyway, but it's still cleaner.